dot/parachain/types/statement.go

```diff
@@ -4,14 +4,18 @@
 package parachaintypes
 
 import (
+	"bytes"
 	"fmt"
+	"io"
 
 	"github.com/ChainSafe/gossamer/lib/common"
 	"github.com/ChainSafe/gossamer/lib/crypto/sr25519"
 	"github.com/ChainSafe/gossamer/lib/keystore"
 	"github.com/ChainSafe/gossamer/pkg/scale"
 )
 
+var BACKING_STATEMENT_MAGIC = [4]byte{'B', 'K', 'N', 'G'}
```

Comment:
```suggestion
var BackingStatementMagic = [4]byte{'B', 'K', 'N', 'G'}
```
https://go.dev/wiki/CodeReviewComments#mixed-caps